src/unity/schemas/Organization.yml

```diff
@@ -1,23 +1,37 @@
 type: object
 properties:
-  idpeId:
-    type: string
-    description: idpe ID of the org
   id:
-    type: number
-    description: quartz ID of the org
+    type: string
+    description: ID of the org
   name:
     type: string
     description: name of the org
-  region:
+  description:
```

Comment:
We haven't added the description to the Quartz DB yet because we weren't confident it was even used, so we might not want to add this just yet. Especially since it will require a trip to IDPE until we move it to Quartz.

Reply:
@eatondustin1 I added to all of the stories; I think we'll take ownership of the field since it exists in IDPE. I added a story to add the field to the table as well.